.github/workflows/push.yml

```
@@ -465,7 +414,8 @@ jobs:
     timeout-minutes: 45
     env:
       AFFECTED_PROJECTS: ${{ matrix.projects }}
-      NX_BASE: ${{ needs.prepare.outputs.NX_BASE }}
+      BASE_SHA: ${{ needs.prepare.outputs.BASE_SHA }}
```

actionlint (1.7.4)

property "base_sha" is not defined in object type {build_chunks: string; cache_key: string; deploy_feature: string; docker_tag: string; images: string; last_good_build_docker_tag: string; node-modules-hash: string; node_image_version: string; test_chunks: string; unaffected: string}

(expression)